### DISCUSSION TITLE
On the meta box should we support more social channels

### DISCUSSION THREAD

**Discussion by AmnestyAM** [amnestywebsite/humanity-theme#47] [Enhancement Requests]
Requested email and WhatsApp

![image](https://github.com/amnestywebsite/humanity-theme/assets/37809410/c79bc371-47b3-4f48-aa00-1df8f6b22569)

**Comment by AmnestyAM** [comment]
https://github.com/amnestywebsite/humanity-theme/discussions/47

**Comment by SophieAmnesty** [comment]
we've had a request for Linkedin on the share icons: 

> Zaira:
When looking at the amnesty website there is no Linkedin icon to share a report on this site (only Facebook and 'Twitter'). See here an example: [Extraction Extinction: Why the lifecycle of fossil fuels threatens life, nature and human rights - ](https://www.amnesty.org/en/documents/pol30/0438/2025/en/)Do you know why is this so outdated? in any case, how do I share a report link on Linkedin? Thank you!!

**Comment by SophieAmnesty** [comment]
also we really should update twitter icon to x

**Comment by SophieAmnesty** [comment]
we had a request about this from Daniela at AI Slovakia: https://github.com/amnestywebsite/amnesty-wp-theme/issues/3056
